src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -66,18 +114,16 @@ public static Phone parsePhone(String phone) throws ParseException {
     }
 
     /**
-     * Parses a {@code String address} into an {@code Address}.
-     * Leading and trailing whitespaces will be trimmed.
-     *
-     * @throws ParseException if the given {@code address} is invalid.
+     * Parses a {@code String comment} into a {@code Comment}
+     * Leading and trailing whitespaces will be trimmed
+     * @return an "-" comment if no comment is added
      */
-    public static Address parseAddress(String address) throws ParseException {
-        requireNonNull(address);
-        String trimmedAddress = address.trim();
-        if (!Address.isValidAddress(trimmedAddress)) {
-            throw new ParseException(Address.MESSAGE_CONSTRAINTS);
+    public static Comment parseComment(String comment) {
+        if (comment == null) {
```

Comment:
is it be better to use comment.trim().isEmpty() instead of comparing it to null?

Reply:
There's a possibility that comment may be a null object (since it's an optional field) and .trim() method exists for String class so it wouldn't work if comment is a null object :P